addons/point_of_sale/tests/test_pos_basic_config.py

```diff
@@ -889,6 +889,27 @@ def open_and_check(pos_data):
         open_and_check(pos01_data)
         open_and_check(pos02_data)
 
+    def test_pos_orders_sequencing(self):
```

Comment:
Could you change the test name ? You are testing the pos_session sequencing, so I would call it something like `test_pos_session_name_sequencing`